### ISSUE TITLE
Does batchnorm converting work properly?

### ISSUE THREAD

**Issue by mrlzla** [nerox8664/pytorch2keras#13]
First of all thanks for this amazing converter, great jog ;)
So I have latest versions of pytorch, keras and this converter. And it seems like batch normalization layer isn't converted properly. Do you know why? Do you have the same issue?)

**Comment by mrlzla** [comment]
Ok. I think it's ok when use:
``` 
for m in model.modules():
    m.training = False
```
